AmericanMedicalAssociation/ama-style-guide-2
# EWL-5810 hamburger menu

## JIRA Ticket(s)
- [EWL-5817: A1 | Theme Global Header hamburger menu](https://issues.ama-assn.org/browse/EWL-5817)


## Description:
Style hamburger menu for global header

## To Test:
-  `gulp serve`
- visit http://localhost:3000/?p=organisms-main-navigation
- observe the global hamburger menu still works
- refer to PR https://github.com/AmericanMedicalAssociation/ama-d8/pull/799 for further testing

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/feature/EWL-5810-global-header/html_report/index.html).


## Relevant Screenshots/GIFs:
<img width="1436" alt="screen shot 2018-09-25 at 4 00 56 pm" src="https://user-images.githubusercontent.com/2271747/46042910-376c9080-c0dc-11e8-9df1-99c9f58b9a67.png">


## Additional Notes:
this PR is linked to a PR in D8 
https://github.com/AmericanMedicalAssociation/ama-d8/pull/799


---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)
